api/agentpb/agent.proto

```diff
@@ -526,11 +526,19 @@ message JobProgress {
   // MySQLRestoreBackup contains restore backup job status update.
   message MySQLRestoreBackup {}
 
+  // Logs contains generic logs from job.
+  message Logs {
+    uint32 chunk_id = 1;
+    string message = 2;
+    bool done = 3;
+    google.protobuf.Timestamp time = 4;
```

Comment:
Why do we need a timestamp here? It's time where we collected logs, but how we can use this information?

Reply:
By the way, we have timestamp on `JobProgress` message level.

Reply:
Reflected this comment
https://github.com/percona/pmm/pull/775#issuecomment-919083768
for now, it's not used

Reply:
But do you have a potential use case for that? We know message timestamp and logs have their own timestamp. I can't find a scenario when we need time when we collected logs.

It's easy to add this field later, if there are no use-cases for now then I think it's better to remove it.